apps/devhub/tests/test_views.py

```diff
@@ -2871,6 +2895,26 @@ def test_unlisted_addon_pass_validation(self, mock_sign_file):
         assert file_.status == amo.STATUS_LITE
         assert mock_sign_file.called
 
+    @mock.patch('devhub.views.sign_file')
+    def test_experiments_are_auto_signed(self, mock_sign_file):
+        """Experiment extensions (bug 1220097) are auto-signed."""
+        self.upload = self.get_upload('experiment.xpi')
+        self.addon.update(guid='experiment@xpi', is_listed=True,
+                          status=amo.STATUS_PUBLIC, trusted=False)
+        # We're going to sign even if it has errors/warnings.
+        self.upload.update(
+            validation='{"notices": 2, "errors": 1, "messages": [],'
```

Comment:
Yep, I'll do that (it was a copy/paste from other tests doing the same thing, I'll change it there too).